Discord Chat Bot Lesson (#1)
hytechclub/py-201: DiscordBot/StudentDesc.md

# Creating a Discord Bot in Python

> Learn how to create a Discord bot in Python, using the [`discord.py`](https://pypi.org/project/discord.py/). Review the **Chat Bot Introduction PowerPoint Presentation** below. Then, complete the **Discord Chat Bot Code-Along Activity**. Once that's done, work on the **Discord Chat Bot Challenges**.

Reviewer: ```suggestion
Learn how to create a Discord bot in Python, using the [`discord.py`](https://pypi.org/project/discord.py/) library. Review the **Chat Bot Introduction PowerPoint Presentation** below. Then, complete the **Discord Chat Bot Code-Along Activity**. Once that's done, work on the **Discord Chat Bot Challenges**.
```

## Chat Bot Introduction PowerPoint Presentation
<iframe src='https://view.officeapps.live.com/op/embed.aspx?src=https://hylandtechclub.com/py-201/DiscordBot/ChatBotIntro.pptx' width='100%' height='450px' frameborder='0'></iframe>

## Discord Chat Bot Code-Along Activity

> Follow the instructions on [this page](DiscordCodeAlong.md) to create a simple Rock Paper Scissors game in Python.

Reviewer: ```suggestion
Follow the instructions on [this page](DiscordCodeAlong.md) to create a simple Chat bot for Discord.
```
